src/components/SimpleModal/SimpleModal.jsx

```diff
@@ -54,8 +55,10 @@ function SimpleModal({
   }
 
   function handleOverlayKeyDown(e) {
+    // Prevent keyboard events originating from the modal from leaking out onto the page.
```

Comment:
@tinkertrain With your solution...would this change still be needed or can we revert it?

Reply:
Hmm, good question. There's a clear case for halting propagation with `esc`. If we think about the modal as a "scoped context" in a page, perhaps your change should stay, I can't came up with a scenario where this would be trouble so my hunch is to keep it.

Let's bring the big guns... @plbabin any thougths?